app/src/main/java/chat/rocket/android/layouthelper/chatroom/RoomListManager.java

```diff
@@ -126,6 +133,40 @@ private void onItemClick(View view) {
     }
   }
 
+  private void updateRoomsList() {
+    removeDeletedItem(unreadRoomsContainer, roomSubscriptionList);
+    removeDeletedItem(channelsContainer, roomSubscriptionList);
+    removeDeletedItem(dmContainer, roomSubscriptionList);
+
+    for (RoomSubscription roomSubscription : roomSubscriptionList) {
+      String name = roomSubscription.getName();
+      if (TextUtils.isEmpty(name)) {
+        continue;
+      }
+
+      String type = roomSubscription.getType();
+
+      if (unreadRoomMode && roomSubscription.getUnread() > 0) {
```

Comment:
Downside to that is isAlert only surfaces the ones we are mentioned in right?  If unread mode for sure want to show them all.  Now if we put mentioned at the very top... That wouldn't hurt anyone's feelings I don't think